src/tests/JIT/HardwareIntrinsics/X86/Avx512DQ_VL_Vector128/Program.Avx512DQ_VL_Vector128.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace JIT.HardwareIntrinsics.X86._Avx512DQ_VL_Vector128
+{
+    public static partial class Program
```

Comment:
Is this necessary?

Reply:
It exists for all the tests and is used to make it much simpler to do various local tweaks and configurations.